### DISCUSSION TITLE
Can we make the icon into a png with transparent whites please?

### DISCUSSION THREAD

**Discussion by kneelesh48** [danny-avila/LibreChat#2386] [Feature Requests & Suggestions]
I'm using the app as a PWA on my Mac and it looks out of place among other icons in the dock

<img width="751" alt="Screenshot 2024-04-11 at 10 17 43 AM" src="https://github.com/danny-avila/LibreChat/assets/25109737/dca7aba7-830e-41ea-aac5-69cda6322a88">

If we can make the whites transparent, it would look much better. Please give it a thought

**Comment by fuegovic** [comment]
We just need to find out what icon pwa on mac is using and we should be able to provide it

**Comment by kneelesh48** [comment]
This shouldn't take much effort. Any plans to work on this @danny-avila?

**Reply by fuegovic** [reply]
Do you know which specific one is used by macOS? I already have a bunch of icon of different resolutions, with and without background, but I haven't had the time to get info on which one macOS is using and don't have an Apple computer to test.

**Reply by kneelesh48** [reply]
How do I find out? I just opened the site in chrome and then installed it as a web app

**Reply by kneelesh48** [reply]
This might help @fuegovic 

<img width="558" alt="Screenshot 2024-04-19 at 9 30 04 AM" src="https://github.com/danny-avila/LibreChat/assets/25109737/39ccd958-5988-4d75-9912-387e615602f0">

**Comment by danny-avila** [comment]
I think this open PR solves this: https://github.com/danny-avila/LibreChat/pull/2366

**Reply by fuegovic** [reply]
> I think this open PR solves this: #2366

I think you're right!

**Comment by kneelesh48** [comment]
This issue has been addressed. This is what the dock looks like now
<img width="795" alt="Screenshot 2024-04-22 at 11 00 52 PM" src="https://github.com/danny-avila/LibreChat/assets/25109737/de93282f-57b7-4ef2-987a-e37ca53ce27e">

Thank you @danny-avila @davecrab
